### DISCUSSION TITLE
Obytes starter changes and migration to the react native 0.76 (using the new architecture )

### DISCUSSION THREAD

**Discussion by shiva-atpar** [obytes/react-native-template-obytes#383] [Ideas] [answered]
Well with react native going for the new architecture and also the performance upgrades we are and allowing to push native changes and with that being mmkv can be upgraded 3.0 allowing for better experience . 
i love Obytes starter for the life i would love to contribute in any way and learn from my fellow devs

**Comment by yjose** [comment, marked as answer]
Hey @shiva-atpar, thank you for considering contributing to the project. We're glad you found it helpful.
As the project is based on Expo, we are waiting for the next stable version to migrate. If you are willing to help, you can open a PR with the migration to the new Expo SDK preveiw version for testing. We will merge it whenever the SDK is stable.
